packages/components/select/src/types.ts

```diff
@@ -35,7 +35,9 @@ export const selectProps = {
   dataSource: IxPropTypes.array<SelectData>(),
   disabled: IxPropTypes.bool.def(false),
   empty: IxPropTypes.oneOfType([String, IxPropTypes.object<EmptyProps>()]),
-  maxLabelCount: IxPropTypes.number.def(Number.MAX_SAFE_INTEGER),
+  maxLabelCount: IxPropTypes.oneOfType([IxPropTypes.number, IxPropTypes.oneOf(['responsive'])]).def(
```

Comment:
```suggestion
  maxLabelCount: IxPropTypes.oneOfType([IxPropTypes.number, IxPropTypes.oneOf(['responsive'])]).def(
  maxLabel: IxPropTypes.oneOfType([IxPropTypes.number, IxPropTypes.oneOf(['responsive'])]).def(
```

这里干脆直接改成 maxLabel 吧，然后保留 maxLabelCount， 在代码里做一下兼容，给个 logger，参考一下其他废弃的 api, 文档里 把 maxLabelCount 改成 maxLabel